fearless/Modules/Staking/SelectedValidators/SelectedValidatorsViewController.swift

```diff
@@ -35,24 +39,39 @@ final class SelectedValidatorsViewController: UIViewController {
 
     private func updateHeaderView() {
         if let viewModel = viewModel {
-            let languages = localizationManager?.selectedLocale.rLanguages
+            let languages = selectedLocale.rLanguages
             let title = R.string.localizable
-                .stakingSelectedValidatorsCount(
+                .stakingSelectedValidatorsCount_v191(
                     "\(viewModel.itemViewModels.count)",
```

Comment:
Better to put whole string into view model to be able to apply formatters

Reply:
Although I cannot see any situations, where we would apply formatters to simple counting integer numbers, the point is valid. Updated view model